examples/annotate-apply-time-mutations-simple/README.md

```diff
@@ -0,0 +1,29 @@
+# annotate-apply-time-mutations: Simple Example
+
+### Overview
+
+This example shows how the [`annotate-apply-time-mutations`] function works.
```

Comment:
You want to link it to the function? This reference as it stands doesn't work. You need to add the link at the end of the page. [example](https://raw.githubusercontent.com/GoogleContainerTools/kpt-functions-catalog/master/examples/create-setters-simple/README.md)